core/tests/test_images.py

```diff
@@ -120,3 +120,23 @@ def test_image_save():
 
     image.save(save_path)
     assert_action_performed_with(image, "save", path=save_path)
+
+def test_pil_support():
+    from PIL import Image as PIL_Image
+
+    pil_img = PIL_Image.open("resources/toga.png")
+    toga_img = toga.Image("resources/toga.png")
+    toga_img_from_pil_img = toga.Image(pil_image = pil_img)
```

Comment:
I notice this is still using the `pil_image` argument - not sure if this is still on your todo list, but it should be.